app.css

```diff
@@ -6,7 +6,299 @@ box-sizing: border-box;
 box-sizing: border-box;
 }
 
-
 body {
 margin: 0;
 }
+
+.header_container {
+background-image: url(images/Background-waves.jpg);
+background-size: 100% 100%;
+overflow: auto;
+position: relative;
+}
+
+.main__heading {
+  padding: 12rem;
+  text-align: center;
+  display: flex;
+}
+
+.nav-bar__container {
+  margin: auto;
+  max-width: 85%;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  padding-top: 1rem;
+  font-size: 1.5rem;
+}
+
+.nav-bar__item-search {
+  width: 6rem;
+  border: none;
+  height: 1.6rem;
+  font-size: 1.2rem;
+  background: none;
+  color: #1e2329;
+}
+
+::-webkit-input-placeholder {
+   color: #474747;
+   font-family: sans-serif;
+   font-size: 1.4rem;
+}
+
+.nav-bar__item-about {
+  text-decoration: none;
+  color: #1e2329;
+  flex: 0;
+  opacity: .8;
+  padding-right: 1rem;
+}
+
+.nav-bar__item-about:hover {
+  color: white;
+  text-shadow: 1px 1px black;
+}
+
+.nav-bar__item-name {
+  text-decoration: none;
+  color: #1e2329;
+  opacity: .8;
+  flex-grow: 1;
+  margin-top: 0;
+  padding-left: 1rem;
+  font-family: sans-serif;
+}
+
+.nav-bar__item-logo_pic {
+  height: 2rem;
+  border-radius: 50%;
+  opacity: .4;
+}
+
+.nav-bar__item-logo_pic:hover {
+  opacity: 1;
+}
+
+.nav-bar__item-title {
+  position: absolute;
+  top: 12%;
+  left: 10%;
```

Comment:
This absolute positioning is throwing things off at different widths.

Either:

* Use the universal centering https://css-tricks.com/centering-css-complete-guide/
* Use a group with the lower title and don't position this text absolute (this would probably be easier and can remove the height and position: relative on your parent container)